### DISCUSSION TITLE
Two snap jobs

### DISCUSSION THREAD

**Discussion by zarMarco** [zrepl/zrepl#786] [General]
Hi all, on my systems I'm using simple snapshot in local. Can I set two jobs on the same file? For example, I would set snapshot for my home dataset every half hour, but for root dataset every hour

**Comment by kpfleming** [comment]
Yes, you can have as many snap jobs as you want. Just set them to snap the datasets you care about on the schedule you wish.

**Reply by zarMarco** [reply]
Thanks I've solved
